src/Widgets/SliderWidget/SliderWidgetEditingConfig.ts

```diff
@@ -32,10 +36,12 @@ provideEditingConfig(SliderWidget, {
     'slides',
     'autoplay',
     ['controls', { enabled: widget.get('autoplay') }],
+    ['autoplayInterval', { enabled: widget.get('autoplay') }],
```

Comment:
* [ ] FYI We could move the three autoplay options ("show controls" effectively is an autoplay option, too) to a dedicated properties group.
* [ ] FYI With two disabled options the UI looks a bit stuffed. If autoplay is turned off often: Maybe we should hide the controls and interval options